communication/marketing-team/storytelling-resources/blog-guidelines.md

```diff
@@ -14,9 +14,22 @@ Other types of content, like Kubernetes capabilities and tutorials, are better s
 
 ## Submit a Post
 
-The quickest way to get involved is to let the team in [#sig-contribex](https://kubernetes.slack.com/archives/C1TU9EB9S) know that you have an idea for an article.
+The quickest way to get involved is to let the team in [#sig-contribex-comms](https://kubernetes.slack.com/archives/C03KT3SUJ20) know that you have an idea for an article.
+
+To reduce the amount of editing done directly in GitHub, a two-stage approach is highly recommended:
+
+1. Create your proposal draft in [Google Docs](https://docs.google.com/) or
+   HackMD (https://hackmd.io), and ask for a review in
+   [#sig-contribex-comms](https://kubernetes.slack.com/archives/C03KT3SUJ20).
+   This will allow easier editing, especially if major changes or restructuring
+   is needed.
+2. Once you have reflected any feedback in the proposal draft, submit the
+   revised article as a Pull Request. Additional changes can still be made
```

Comment:
So one thing we talked about is having the *editor* actually submit the PR.  The reason for this is setting the date.  @Debanitrkl said there's a way to make sure that the author still gets commit credit.

Reply:
Indeed, I remember that conversation. My goal with this initial PR was to prevent someone from opening PRs in the wrong repository, since the rest seemed to be still being discussed.
If it's agreed that the editor should be the one opening the PR, I can make changes in that direction. @Debanitrkl , is there anything that should be considered in that process? It would make sense to document any action that the editor should take, for example.

Reply:
https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors
Having co-author on the commit is the way to add the author.